<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Messenger</class>
 <widget class="QMainWindow" name="Messenger">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>369</width>
    <height>481</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Messenger</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="sendButton">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>410</y>
      <width>51</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>&gt;</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textInput">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>410</y>
      <width>271</width>
      <height>31</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>Введите текст...</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="messagesBrowser">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>100</y>
      <width>331</width>
      <height>291</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>17</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Skillbox Messenger</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="nameInput">
    <property name="geometry">
     <rect>
      <x>238</x>
      <y>60</y>
      <width>113</width>
      <height>21</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>Введите имя...</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>203</x>
      <y>63</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Имя:</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
